### PR TITLE
WIP: Azure Queues

### PR DESCRIPTION
**Pull request by Henni**
use Azure's ServiceBusService to pass messages.

### CODE REVIEW

**Review by pfent — APPROVED** [review]
Seems fine.

**Review by Sandr0x00 — DISMISSED** [review]
This should not yet be merged ;)

**Comment by ansjin** [comment]
This should not be merged

File: app/config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// config.js
+module.exports = {
+  'queue': {
+    'namespace_access_key': 'IJ+f4X+iauM+ji9Dzjz7ZwZa+E7MUtiulcvv2Cu9A4M=',
+    'namespace': 'relationshipns',
```
Comment (simonzachau) [inline app/config.js:5]: should it be ```relationships```?

Comment (ansjin) [reply app/config.js:5]: nope, its relationshipns only (ns for namespace)

**Review by ansjin — CHANGES_REQUESTED** [review]
not to be merged. I am working on it

**Comment by ansjin** [comment]
Now this can be merged.

**Review by ansjin — APPROVED** [review]

**Comment by kordianbruck** [comment]
@anshul0708 we need your to finish this! Please respond in Gitter!

**Comment by Sandr0x00** [comment]
@kordianbruck Wrong Anshul, we got the **Jindal** not the **Sharma** 😅

**Comment by kordianbruck** [comment]
@Sandr00 wups :disappointed:

**Pull request merged**